js/lib/widget.js

```diff
@@ -32,7 +32,7 @@ var RemoteFrameBufferModel = widgets.DOMWidgetModel.extend({
         // For the frames
         frame_feedback: {},
         // For the widget
-        css_width: '100%',
+        css_width: '500px',
```

Comment:
Your PR description says 600, this says 500.

Reply:
Updated the PR description :)